tango_ros_common/tango_ros_native/src/tango_ros_conversions_helper.cpp

```diff
@@ -278,6 +278,8 @@ void toOccupancyGrid(const Tango3DR_ImageBuffer& image_grid,
       // the occupancy grid is using (x: right, y: up). The image is therefore
       // flipped around the x axis.
       int value = static_cast<int>(image_grid.data[j + (image_grid.height - i - 1) * image_grid.width]);
+      occupancy_grid->data.push_back(value);
+      continue;
       if (value == 1) {
```

Comment:
I would prefer to comment out the code here instead of putting continue to make it very clear that it not used anymore.